### DISCUSSION TITLE
recall last argument

### DISCUSSION THREAD

**Discussion by tkossak** [leo-arch/clifm#204] [Q&A] [answered]
In bash you do it with `alt+.` but clifm has it bound by default to hidden files toggle. Do we have any other shortcut for recalling last arguments (from previous commands)?

**Comment by leo-arch** [comment, marked as answer]
That's a nice suggestion. I'll see what I can do.

**Reply by leo-arch** [reply]
Hey @tkossak! `yank-last-arg` is built into readline, so that it is available by default, but in **clifm** it is overridden by `toggle-hidden`.

To unlock `yank-last-arg` just comment out the `toggle-hidden2:\M-.` line in your keybindings file (`kb edit`), and restart **clifm**.

NOTE: Since this is a readline built-in, **clifm**'s specific features (syntax highlighting and suggestions) will be ignored. For example: inserted arguments won't be highlighted (colored), and current suggestions won't be deleted. To fix this, I need to write a custom implementation of this function. But for the time being, this just does the trick.

**Reply by leo-arch** [reply]
Hi @tkossak. Is this working for you?

**Reply by tkossak** [reply]
It works, thank you
